### PR TITLE
Get Firebase Crashlytics build going

### PR DESCRIPTION
**Pull request by dbnicholson**
The basics for getting Firebase Crashlytics building. This depends on https://github.com/endlessm/python-for-android/pull/11. I'm marking it as a draft just to show what's needed.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 cython
 virtualenv
-git+https://github.com/endlessm/python-for-android@v2022.09.04-endless1#egg=python-for-android
+git+https://github.com/endlessm/python-for-android@8a506eff5261635d987bc9b578a021c7d84d22e5#egg=python-for-android
```
Comment (dbnicholson) [inline requirements.txt]: This is a pointer to https://github.com/endlessm/python-for-android/pull/11. If it works we'd want to merge that PR, make a `v2022.09.04-endless2` tag, and update this accordingly.

**Review by xcho1 — COMMENTED** [review]
added tag and changed requirements.txt

**Comment by dbnicholson** [comment]
I changed the Makefile so that it would install SDK 31 and squashed the commits together with some more background in the commit message.

**Comment by dbnicholson** [comment]
retest this please

**Review by manuq — APPROVED** [review]

**Pull request merged**